crates/rnote-ui/src/appwindow/actions.rs

```diff
@@ -567,13 +565,6 @@ impl RnAppWindow {
             appwindow.handle_widget_flags(widget_flags, &canvas);
         }));
 
-        // New doc
```

Comment:
This action is still needed for the `New` menu item in the app-menu.

Reply:
True. But I don't think it should use the old `new-doc` action, but should also create a new tab. I don't think replacing your current tab is the desired behaviour.

Reply:
Let's keep the action anyway, in case it is needed again at some point in the future.